makejdk-any-platform.1

```diff
@@ -128,6 +132,11 @@ specify any custom user make arguments.
 .BR \-\-no\-adopt\-patches
 This indicates to the build script that the build does not have Temurin's additional patches applied.
 .TP
+.BR  \-o ", " \-\-openjdk\-source " " \fI<dir/archive>\fR
+If set, must point to directory or archive with sources.
+Those will be used instead of cloning without any .git magic.
```

Comment:
It is not jsut clonning. There is git reset or deduction of tags, and a bit of more. The git is called aprox 90times in build scripts.
So what about: `This local source code will be used instead of cloning from a git repository and without any git tag or git reset` ?